cypress/integration/BounceRulesContainer/bounce_rules.spec.js

```diff
@@ -14,4 +14,25 @@ describe("Bounce Rules Page", () => {
     BounceRulesPage.ruleFilter.should("be.visible");
     BounceRulesPage.ruleTable.should("be.visible");
   });
+
+  it("should create a bounce rule", () => {
+    BounceRulesPage.createRuleButton.click();
+    BounceRulesPage.fillForm(
+      "1",
+      "cypressTest",
+      "528",
+      "testDescription",
+      "492",
+      "testRegex"
+    );
+    BounceRulesPage.confirmModal.should("be.visible");
+    BounceRulesPage.confirmationSubmit.click();
+    BounceRulesPage.testRule.should("be.visible");
+  });
+
+  it("should delete a bounce rule", () => {
```

Comment:
Ahh here is an antipattern to have another test to depend on the previous test. Tests in general should be able to work by themselves and not be dependent on one another's success/failure. What I meant in both the creation test and the delete bounce rule tests is to have a teardown/setup before both of these tests through the API. Before the create a bounce rule test we should delete the test bounce rule through the API to be sure it was cleaned up beforehand and then create the bounce rule through the UI and verify the result. For the deletion test, we should be sure to create the specific test rule through the API to be sure it is there and then delete through the UI and verify. We can also have a helper function in the page object like deleteBounceRule() by something like id or some property of the object, chain the final action of that function with .then() and then assert about what should or should not be there similar to how we did the login tests with chaining .then().